src/app/memberships/memberships-edit.component.html

```diff
@@ -0,0 +1,27 @@
+<form [formGroup]="membershipForm" class="memberships-edit-form">
+  <div *ngIf="errorMessage" class="memberships-edit-error">{{errorMessage}}</div>
+  <div fxLayout="column">
+    <div fxFlex>
+      <mat-form-field class="memberships-edit-form-field">
+        <input matInput id="memberships-edit-title" placeholder="Topic or name of the organisation/web-site (for blogs)" formControlName="name" [required]="true">
```

Comment:
I suggest to place here `Organisation, web-site, blog, etc.` instead of `Topic or name of the organisation/web-site (for blogs)`.